RiotSharp/StatusRiotApi.cs

```diff
@@ -6,24 +6,35 @@
 using System.Threading.Tasks;
 using RiotSharp.Misc;
 using System;
+using System.Net.Http;
 
 namespace RiotSharp
 {
-    public class StatusRiotApi : IStatusRiotApi
+    public partial class StatusRiotApi : IStatusRiotApi
```

Comment:
Why do you want to split this into two partial classes? I don't think this is really needed.

Reply:
You can split them up into two different files. However, spliting the class up in the same file doesn't really provide a benefit in my eyes. You can achieve the same effect with regions.